package.json

```diff
@@ -7,9 +7,6 @@
     "Jaroslav Šmolík <grissius@gmail.com>"
   ],
   "main": "./dist/index.js",
-  "engines": {
```

Comment:
maybe instead of removing the engines, we should have range for the tested versions? Sth like this? `">=8 <=14"`